assets/js/theme/global/modal.js

```diff
@@ -181,13 +188,6 @@ export class Modal {
         this.$modal.on(ModalEvents.opened, this.onModalOpened);
     }
 
-    unbindEvents() {
-        this.$modal.off(ModalEvents.close, this.onModalClose);
-        this.$modal.off(ModalEvents.closed, this.onModalClosed);
-        this.$modal.off(ModalEvents.open, this.onModalOpen);
-        this.$modal.off(ModalEvents.opened, this.onModalOpened);
-    }
```

Comment:
Why do we need to removed method which is in use?

Reply:
Unbind is not in use now. It was using firstly in onmodalclosed function then I replaced it to constructor. After we decided to remove it from constructor - it is not in use anymore

Reply:
Why did we move it from onmodalclosed?

Reply:
Because after we closed modal once, we still need this events to exist. For them to work when modal will be opening and closing next times

Reply:
Could we put event binding on modal opened and unbinding on modal closed?